config/settings.yml

```diff
@@ -57,7 +57,7 @@ binaries:
 db_encryption_key: f01ff8ebd1a2b053ad697ae1f0d86adb
 
 database_url: postgis:///vets-api
-test_database_url: postgis:///vets-api-test
+test_database_url: postgis://postgres:password@postgres:5432/vets_api_test
```

Comment:
is this change absolutely necessary?  I'll require everyone to create this db user with this publically available password and many/most people  don't use docker.

Reply:
So it needs to match the `test_database_url` pattern in the `docker_compose.yml` file. But, I had a thought. What If I left this line unchanged, but in the `README` told potential users to add `test_database_url: postgis://postgres:password@postgres:5432/vets_api_test` in `settings.local.yml`. A bit tacky, but not end of the world.

Reply:
Actually, @annaswims, mind reviewing the latest commit? It turns out, when I use the command `parallel_tests` it lets me run a command in parallel while persisting the environment settings (in this case, specifically the Settings override from the docker-compose file). So I'm able to just create the databases with that command, and it appears to work. Doing this, I don't need to change `Settings.yml` at all!